frontend/app/model/RichEvent.scala

```diff
@@ -108,7 +108,8 @@ object RichEvent {
     }
 
     //TODO: Should be Option[String]
-    val socialImgUrl = imgOpt.map(_.defaultImage).getOrElse("")
+    //TODO in separate PR make this an option and update views
+    val socialImgUrl = imgOpt.flatMap(_.defaultImage).fold("")(d => d)
```

Comment:
This has also been sorted recently in `master`, it's an option now. Might want to rebase to get the latest changes.